src/dash-playlist-loader.js

```diff
@@ -43,7 +43,7 @@ export const parseMasterXml = ({ masterXml, srcUrl, clientOffset, sidxMapping })
     sidxMapping
   });
 
-  addPropertiesToMaster(master, srcUrl);
+  addPropertiesToMaster(master, srcUrl, 'dash');
```

Comment:
I think this should be `true` instead of `'dash'`

Reply:
oops, I made the change locally but apparently forgot to save.